canonical-webteam/deployment-configs
# Increase proxy-buffer-size in nginx configuration

Fixes https://github.com/canonical-websites/snapcraft.io/issues/209

When we request https://login.ubuntu.com the default configuration of ingress-nginx blocks it because the headers are too big (we get a 502).

We need to discuss about the name we want to give to the file... I don't really want to put this piece of code in the file `namespaces.yaml`

# QA
- Checkout to this branch
- `minikube start`
- `sudo vim /etc/hosts` -> add line snapcraft.io and ip : `minikube ip`
- `./qa-deploy snapcraft.io`
- Wait that snapcraft.io is
- `kubectl apply -f config.global.yaml`
- access https://snapcraft.io/account you should be redirected to login.ubuntu.com
